proj/src/main/webapp/script.js

```diff
@@ -205,3 +203,19 @@ function clickWithEnterKey() {
         }
     });
 }
+
+function createMessageWithTemplate(messageObj) {
+    const messageTemplate = document.getElementById('message-temp');
+    const message = messageTemplate.content.cloneNode(true);
+
+    const msgHeader = message.querySelector('.message-header');
+    msgHeader.getElementById('username').innerText = 'name go here';
+            
+    const msgBody = message.querySelector('.message-body');
+    msgBody.innerText = messageObj.content;
+    return message;
+}
+
+function getPath(tag, chatId) {
```

Comment:
to keep using the test tag, you can override tag to be 'test' in here so that you don't need to put `getPath('test',CHAT_ID)` everywhere and you can just create placeholder variables for tag where you are calling this.

Reply:
what if instead of returning the path this function just returned the reference? that way it kills two birds with one stone

Reply:
That also works really well.